doc/availability.md

```diff
@@ -211,4 +211,4 @@ Let’s imagine a situation where the service has imposed a limit on 2000 series
 The use case can be supported as follows:
 
 1. When the user adds or removes a data query filter by checking or unchecking a checkbox, call the Data Availability API with the current data query state.
-2. The response will include the Cube Region constraint. If the Cube Region constraint has an Annotation with type `sdmx_metrics` and an Id of `series_count` then obtain the count via the Annotation's title. Use this count to either enable or disable the 'Run Query' button.
+2. The response will include the DataAvailability constraint. If the DataAvailability constraint has a seriesCount it can be used to either enable or disable the 'Run Query' button based on its value.
```

Comment:
Everywhere in the document, the term `AvailabilityConstraint` is used, except in this paragraph where `DataAvailability constraint` is used instead. Is there any difference between the two? This might be correct, but I thought it was better to ask 😊

Reply:
Good spot, no difference, I've changed it to AvailabilityConstraint.